.babelrc

```diff
@@ -1,13 +1,105 @@
 {
   "env": {
     "development": {
-       "presets": ["env", "stage-1"]
-     },
+      "presets": [
```

Comment:
At some point I think we could pare these down because I know we don't use a lot of these things (e.g. decorators). Just don't think we want to needlessly increase the file size of the bundled app or the amount of deps that have to come down the wire.